EMCAL/EMCALrec/AliEMCALClusterizerv3.h

```diff
@@ -0,0 +1,122 @@
+#ifndef ALIEMCALCLUSTERIZERV3_H
+#define ALIEMCALCLUSTERIZERV3_H
+
+/* Copyright(c) 1998-1999, ALICE Experiment at CERN, All rights reserved. *
+ * See cxx source for full Copyright notice                               */
+
+
+#include "AliEMCALClusterizerv1.h"
+class AliEMCALRecPoint; 
+class AliEMCALDigit;
+
+// Define numbers rows/columns for topological representation of cells
+const Int_t kNrows    = (24+1)*(6+4);  // 10x supermodule rows (6 for EMCAL, 4 for DCAL). +1 accounts for topological gap between two supermodules
+const Int_t kNcolumns = 48*2+1;        // 2x  supermodule columns + 1 empty space in between for DCAL (not used for EMCAL)
+
+struct cellWithE {
```

Comment:
Sorry, I overlooked this in the first round: Since the struct is only used in AliEMCALClusterFinder I would make this a private nested struct of the AliEMCALClusterFinder and not expose it to the public.

Reply:
Thanks, this is true. I have also put the const definitions into a namespace.